capstone/src/main/java/com/google/sps/servlets/VoterServlet.java

```diff
@@ -35,11 +38,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     URLConnection connection = new URL(url + "?" + query).openConnection();
     connection.setRequestProperty("Accept-Charset", charset);
     InputStream result = connection.getInputStream();
+    Scanner scanner = new Scanner(result);
+    String responseBody = scanner.useDelimiter("\\A").next();
+    JSONObject obj = new JSONObject(responseBody);
+    String election = obj.get("election").toString();
+    JSONObject election_obj = new JSONObject(election);
```

Comment:
nit: this logic can be replaced by fewer lines of code:
JSONObject obj = new JSONObject(scanner.useDelimiter("\\A").next());
JSONObject election_obj = new JSONObject(obj.get("election").toString());
nit: Java naming convention is Camel Case:
JSONObject responseJsonObject = ...
JSONObject electionJsonObject = ...

Reply:
done